### PR TITLE
Basic UI: slider with value

### LINKED ISSUE #3369: Basic UI: display value for dimmer items
https://community.openhab.org/t/basicui-and-slider-item-no-values-visable/11406

The wish is to optionally display the value for a dimmer item, when a format is set in the item label.

Classic UI is doing that, not Basic UI.

### PR DESCRIPTION
**Pull request by lolodomo**
Fix #3369
Add support for label and value color

Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

**Comment by lolodomo** [comment]
Testing my change, I discovered that sometimes the value is not updated or even wrong. It does not happen when dragging the slider but only when clicking on the slider. It is not systematic. I really don't think that the problem is due to my change.There are already existing issues relative to slider so I imagine I just encountered one of them.

**Comment by lolodomo** [comment]
@resetnow : can you review please ?

File: extensions/ui/org.eclipse.smarthome.ui.basic/web-src/smarthome.less [hunk 1]
```diff
@@ -371,6 +371,9 @@
 		&--switch {
```
Comment (resetnow) [inline extensions/ui/org.eclipse.smarthome.ui.basic/web-src/smarthome.less]: Please combine the selectors:
```less
&--switch, &--slider {
...
```

File: extensions/ui/org.eclipse.smarthome.ui.basic/web-src/smarthome.less [hunk 2]
```diff
@@ -371,6 +371,9 @@
...
 			padding: 0 10px 0 10px;
 		}
+		&--slider {
+			padding: 0 10px 0 10px;
+		}
```
Comment (resetnow) [inline extensions/ui/org.eclipse.smarthome.ui.basic/web-src/smarthome.less]: This can be removed.

**Comment by lolodomo** [comment]
Less file updated. Thank you again @resetnow for your quick review.
Ready for merge.

**Pull request merged**